README.md

```diff
@@ -23,12 +23,7 @@ import Ember from 'ember';
 import StateMixin from 'ember-state-services/mixin'
 
 export default Ember.Object.extend(StateMixin, {
-  stateName: 'emailEdit',
-  setupState: function(Factory, model) {
```

Comment:
hahaha indeed. So our readme example should show you how to use it, right? `stateName` is required, but `setupState` has a [default implementation](https://github.com/stefanpenner/ember-state-services/blob/master/addon/mixin.js#L20) so why imply they need to reimplement exactly that? They may need to override it to do some sort of other behavior on top of the default, sure, but this example does not show that. An example showing that probably should call `_super()` to get the instance (keeping container reference) then do whatever else needs to be done.